src/Profilers/MonitorProfiler/MainProfiler/MainProfiler.cpp

```diff
@@ -283,6 +283,10 @@ HRESULT MainProfiler::ProfilerCommandSetCallback(const IpcMessage& message)
     {
     case ProfilerCommand::Callstack:
         return ProcessCallstackMessage();
+    case ProfilerCommand::Start:
```

Comment:
Per Joe/Justin:
- Do more testing around app "death" instead of dotnet-monitor death.